ui-v2/src/components/ui/flex.tsx

```diff
@@ -0,0 +1,20 @@
+import { cn } from "@/lib/utils";
+import { createElement, forwardRef } from "react";
+import { UtilityProps, spacingUtiltiesClasses } from "./utils/spacing-utils";
+
+type Props = UtilityProps & {
```

Comment:
Similarly, we might want to omit `display` here so that this always stays `flex`.